firewall requirements

PedroPerezMSFT opened the pull request:
testing on networks with restrictions requires opening certain ports. Suggested clarification at the bottom of the document

azurecla commented:
Hi __@arunvinoth__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

davidwrede commented:
Thanks for the contribution, @arunvinoth!

PedroPerezMSFT committed:
firewall requirements

testing on networks with restrictions requires opening certain ports. Suggested clarification at the bottom of the document

steveesp reviewed (approved):
Thanks, Pedro!
